ex5/ex5.sh

```diff
@@ -0,0 +1,26 @@
+# 1
+for letter in {A..F}; do touch "file${letter}.txt"; done
+# 2
+for letter in {C..F}; do rm "file${letter}.txt"; done
+# 3
+mv fileA.txt fileA.old
+# 4
+find ./ -type f -delete # find looks for the the files in subs dir
+                        # -type f means to look for files
+                        # instructs to delete
+# 5
+cp -ri /etc/*.conf ./ 
+# 6
+cat /etc/l*
+# 7
+find /path/to/directory -type f -name '????' ! -name '*.conf' -exec grep -l 'user' {} +
```

Comment:
this works, but you could do something much simpler in this fashion: 
`grep -l "user" ????.conf`